SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/EditorWorkerController.h

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma once
+
+#if WITH_EDITOR
+
+#include "Editor.h"
+#include "Settings/LevelEditorPlaySettings.h"
+
+namespace improbable
```

Comment:
I know we had a conversation about this a long time ago, but when do we put stuff under namespace improbable?

Reply:
When wrapping non Unreal derived types.